source/VirtualDesktop/VirtualDesktop.cs

```diff
@@ -86,5 +86,35 @@ public VirtualDesktop GetRight()
 				return null;
 			}
 		}
+
+#region IDisposable
+		private bool disposed = false;

```

Comment:
```suggestion
		private bool _disposed = false;
```

Use prefix `_` for the fields.